addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -122,12 +126,7 @@ def reset_efaktur(self):
         return True
 
     def download_csv(self):
-        action = {
-            'type': 'ir.actions.act_url',
-            'url': "web/content/?model=ir.attachment&id=" + str(self.l10n_id_attachment_id.id) + "&filename_field=name&field=datas&download=true&name=" + self.l10n_id_attachment_id.name,
-            'target': 'self'
-        }
-        return action
+        return self.l10n_id_efaktur_document.download_attachment()
 
     def download_efaktur(self):
```

Comment:
>                     raise ValidationError(_('E-faktur is not available for invoices without any taxes.'))
This error is unclear; I received it when trying to download efaktur on an invoice with a tax.
The issue was that the partner was not set as PKP

> Connect INV/2024/00005 with E-faktur to download this report

It is the same thing with this one. How can the user know that connect means 'reset to draft and post again'?
I know it's not related to your changes, but we can improve it.

Reply:
you're right , think there has been multiple questions before from FCs and customers of this misleading error being thrown